pkg/logger/logger.go

```diff
@@ -11,7 +11,23 @@ import (
 )
 
 var (
-	pionLevel zapcore.Level
+	pionLevel           zapcore.Level
+	pionIgnoredPrefixes = map[string][]string{
```

Comment:
any concerns with suppressing these? they didn't appear to be useful.

Reply:
No concerns